core/src/main/java/org/pentaho/platform/web/http/api/resources/exception/FallBackSchedulerException.java

```diff
@@ -0,0 +1,18 @@
+
+package org.pentaho.platform.web.http.api.resources.exception;
```

Comment:
Since `FallBackSchedulerException.java` extends `SchedulerException.java` the package name should also be an extension too.

Also, if there needs to be some reflection logic or exclusion, it can be done with a simple regex.

So the fully qualified name could be
- org.pentaho.platform.api.scheduler2.FallBackSchedulerException
- org.pentaho.platform.api.scheduler2.fallback.FallBackSchedulerException